rock_paper_scissors.py

```diff
@@ -0,0 +1,61 @@
+#!/bin/python3
+print('''Let\'s play a game.
+''')
+def playgame():
+  player = input('Rock (1), paper (2) or scissors (3)?')
+  player = int(player)
+  if player ==1:
+    print ('You choose rock.')
+  elif player ==2:
+    print ('You choose paper.')
+  elif player ==3:
+    print ('You choose scissors.')
+  else:
+    print ('Please type 1, 2, or 3.')
+    playgame()
+  import time
+  time.sleep(1)
+  print('''Nice.
+  ''')
+  time.sleep(2)
+  print('1...')
+  time.sleep(1)
+  print('2...')
+  time.sleep(1)
+  print('''3!
+  ''')
+  time.sleep(1)
+  from random import randint
+  chosen = randint(1,3)
+  if chosen==1:
+    print('Your opponent chooses rock.')
+  elif chosen==2:
+   print('Your opponent chooses paper.')
+  elif chosen==3:
+    print('Your opponent chooses scissors.')
+  time.sleep(1)
+  if player==chosen:
+    print('''It was a draw.
+    ''')
+    victory=2
+  if (player==1 and chosen==2) or (player==2 and chosen==3) or (player==3 and chosen==1):
```

Comment:
This `if` statement has enough conditions that I would split the decision into a separate function:

```
def is_victory(player, computer):
    # rock beats scissors
    if player == ROCK and computer == SCISSORS:
        return True
    # paper beats rock
    if player == PAPER and computer == ROCK:
       return True
    # scissors beats paper
    if player == SCISSORS and computer == PAPER:
        return True
    # rely on a draw being caught elsewhere
    return False

if player == chosen:
    print("You drew...")
elif is_victory(player, chosen):
    print("You won!")
else:
   print("You lost!")
```